src/js/components/AppListItemComponent.jsx

```diff
@@ -340,6 +360,26 @@ var AppListItemComponent = React.createClass({
     );
   },
 
+  getServiceLink: function () {
+    var model = this.props.model;
+    var scheme = (model.labels && "SERVICE_SCHEME" in model.labels &&
+       (model.labels["SERVICE_SCHEME"] === "http" ||
+        model.labels["SERVICE_SCHEME"] === "https"))
```

Comment:
at the moment we dont have any service exposed on https (but soon!)

Reply:
We won't have to bother then :).